tests/perf/mt_ubench_l3.cpp

```diff
@@ -46,7 +46,7 @@ int main(int argc, char** argv) {
             assert(e == 0 || e == PTHREAD_BARRIER_SERIAL_THREAD);
 #endif
             for (int j = 0; j < L3_PERF_UBM_NMSGS; j++) {
-                l3_log_fast("Hello, world! %d %d", 0, j);
+                l3_log("Hello, world! %d %d", 0, j);
```

Comment:
This is nice! As a user, I don't need to remember fast or slow, and the .s version is just picked up if it exists for the current platform. Hope this simplifies adoption in months to come ...

Reply:
thanks!